packages/server/src/api/controllers/static/templates/BudibaseApp.svelte

```diff
@@ -95,7 +95,7 @@
   </style>
 </svelte:head>
 
-<body id="app">
```

Comment:
This change was necessary to render multiple components in the app.hbs template body, the location of the client app main svelte entry point remains the same, so it hopefully won't break any tests.

Reply:
The issue was that it was using body, or that we had duplicated ids?